src/main/java/edu/kis/powp/jobs2d/drivers/visitor/DriverDetailsVisitor.java

```diff
@@ -0,0 +1,49 @@
+package edu.kis.powp.jobs2d.drivers.visitor;
+
+import edu.kis.powp.jobs2d.commons.Pair;
+import edu.kis.powp.jobs2d.drivers.*;
+import edu.kis.powp.jobs2d.features.RecordFeature;
+
+public class DriverDetailsVisitor implements DriverVisitor {
+    private StringBuilder stringBuilder = new StringBuilder();
+    public void clear() { stringBuilder = new StringBuilder(); }
+    public String getDetails(IDriver driver) {
+        clear();
+        driver.accept(this);
+        return stringBuilder.toString();
+    }
+
+    @Override
+    public void visit(LoggerDriver loggerDriver) {
+        Pair<Integer, Integer> position = loggerDriver.getPosition();
+        String details = "\nLoggerDriver details:" +
+                "\n Position: (" + position.getFirst() + "," + position.getSecond() + ")";
+        stringBuilder.append(details);
+    }
+
+    @Override
+    public void visit(DriversComposite driverComposite) {
```

Comment:
DriversComposite consists of other drivers. Details should also contain information generated by DriverDetailsVisitor also for those drivers. You should use, e.g., for loop with accept (to perform DFS visitation).